components/giftbox.vue

```diff
@@ -170,6 +170,10 @@
 
   .modal-content {
     max-height: 95vh;
+    -webkit-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+    -moz-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+    -o-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+    box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
```

Comment:
Nit: I don't think we need the browser prefixes for `box-shadow` anymore. See https://css-tricks.com/do-we-need-box-shadow-prefixes/ and https://caniuse.com/#feat=css-boxshadow

Worst case scenario someone with a _very_ old browser won't see the shadow, but it won't hurt the experience.